docs/tm/udq_actionx/pyaction.tex

```diff
@@ -226,11 +226,11 @@ \section{Changing the \inlinecode{Schedule} object - using a ``normal'' \actionx
 ENDACTIO
 \end{deck}
 The \inlinecode{CLOSEWELLS} action is set up to run zero times, so the normal
-\actionx{} machinery will never run this action\footnote{The condition
-\inlinecode{DAY=1} is completely dummy, in the future the \actionx{}
-implementation should be improved to handle empty conditions.}. Then in the
-Python run function we go through all the wells and call the
-\inlinecode{CLOSEWELL} action to close those with \inlinecode{OPR < 1000}:
+\actionx{} machinery will never run this action\footnote{The \kw{CLOSEWELL}
+action has an \emph{empty condition}, the \actionx{} keywords with empty
+condition will always evaluate as false.}. Then in the Python run function we go
+through all the wells and call the \inlinecode{CLOSEWELL} action to close those
+with \inlinecode{OPR < 1000}:
```

Comment:
Looking at this change, I am wondering what happens if the user puts a condition here (like before the change).  Maybe a comment about that would be good?

Reply:
OK - I have rephrased this quite a lot.